packages/features/embed/Embed.tsx

```diff
@@ -578,6 +583,19 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
   const defaultConfig = {
     layout: BookerLayouts.MONTH_VIEW,
   };
+
+  const paletteDefaultValue = useCallback(
+    (paletteName: string) => {
+      if (paletteName === "brandColor") {
+        const color = hasDarkTheme ? defaultBrandColor.darkBrandColor : defaultBrandColor.brandColor;
```

Comment:
Even though we decide dark/light based on current theme, user is still not able to test on dark theme because whatever color is chosen applies for light theme only as the styles API we are using currently only support light theme.

We should switch to using cssVarsPerTheme as shown in codesandbox demo

Reply:
See this loom that shows the issue https://www.loom.com/share/5ebab54eadb54dd5917d11a42812b8ed